### PR TITLE
Remove CPU_tensor_apply* from Normalization.cpp

### PR DESCRIPTION
**Pull request by ifedan**
https://github.com/pytorch/pytorch/issues/24486
https://github.com/pytorch/pytorch/issues/24485
https://github.com/pytorch/pytorch/issues/24484
https://github.com/pytorch/pytorch/issues/24483
https://github.com/pytorch/pytorch/issues/24482
https://github.com/pytorch/pytorch/issues/24481
https://github.com/pytorch/pytorch/issues/24480
https://github.com/pytorch/pytorch/issues/24479

### CODE REVIEW

**Comment by ifedan** [comment]
image

**Comment by ifedan** [comment]
```python
import torch
import timeit
import itertools
import statistics

for size in [(4, 3), (10, 10), (100, 100), (1000, 1000)]:
    for train in [True, False]:
        f = size[1]
        setup_code = '''import torch; import torch.nn as nn; module = nn.BatchNorm1d({0});'''
        if train:
            setup_code += '''module.eval();'''
        setup_code += '''data = torch.rand({1})'''
        setup_code = setup_code.format(f, size)
        test_code = '''res1 = module(data);'''
        result = timeit.repeat(setup = setup_code,stmt = test_code,repeat = 3,number = 100)
        mean = statistics.mean(result)
        std = statistics.stdev(result)
        print('''size = {0} time = {1}'''.format(size, mean))
```

**Review by VitalyFedyunin — APPROVED** [review]
